### ISSUE TITLE
Firefox tabs not shown in processes

### ISSUE THREAD

**Issue by pukkancsanyo** [adirh3/Fluent-Search#337]
Maybe it is just me, but so far Firefox (and Tor) tabs also appeared among the running processes, so I could search for them and switch to them as it is the case with any Chromium-based browser. However, it does not work anymore, Fluent Search can only see the Firefox tab which is open, but nothing more. The feature is still flawless with Chrome, Edge & co.

As it can be seen on the attached screenshot, only the active tab is present among the search results, the other with similar name is not.

I'm on Windows 10 [20H2], Fluent Search version 0.9.88.9989, Firefox 93.0.

![image](https://user-images.githubusercontent.com/89463030/138066144-4f21f79a-ddc8-4790-8096-7668ed58fbec.png)

**Comment by ryanmholmes** [comment]
Adding an observation:

**Comment by adirh3** [comment]
> Thanks a lot for your reply. I tried as suggested, downloading 0.9.87.0. This version seems to work impeccably with the latest Firefox and Tor; however, it does not show up anything with any other hardened Firefox or Librewolf. Oddly, if I change back from a hardened portable profile to the default one with the default user.js, Fluent Search appears to stay perplexed, and can’t see the processes.

Thanks for checking testing!
Lets see if I understand - 
Fluent Search 0.9.87 (Btw why not 88?) -  can find Tabs in FireFox 
Fluent Search 0.9.89 - can find only the focused tab in FireFox
Not sure I understand the part about the portable profile, you mean FireFox profile? Any chance you disabled any accessibility setting in FireFox? Fluent Search uses Windows' accessibility to read UI elements

**Comment by pukkancsanyo** [comment]
Hi,
You understood and summed it up correctly. It also seems to work with 88, but I failed to mention it because I did not test it so extensively.

As for the second part, yes, in Firefox portable you can have multiple profiles (e. g., a vanilla and a hardened one), and you can switch between them. However, after using the hardened one, Fluent Search does not appear to work with the vanilla one either.

Thanks for the tip with accessibility features; I have tried to tweak them but without any notable success so far.

**Comment by pukkancsanyo** [comment]
Hi, I'm sorry for being late with the feedback, the summer was a bit busy.
I tested the new stable, and it seems to work now with FF. Tor also appears to work out of box most of the times with some occasional hiccups I cannot explain (sometimes searching for tabs does not switch to the specific tab itself but to the last opened tab of the browser).
Librewolf also seems to work now with some accessibility tweaking in about:config.
All in all, it's a pleasing experience to use it, thanks for the tips and everything that you've done in this regard.
If I found something later on with more extensive testing, I'll let you know, but so far so good.

Cheers
